Subject: [PATCH] server.c: show correct crypt support result

If the crypt function is available as part of the standard
system library, then HAV_LIBCRYPT will not be set, but
HAVE_CRYPT or HAVE_CRYPT_R will.

Make server.c test HAVE_CRYPT, HAVE_CRYPT_R  and HAVE_LIBCRYPT
to determine the correct value of crypt support.

Signed-off-by: Kyle J. McKay

---
 src/server.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/server.c b/src/server.c
index d79594cd..e096de1b 100644
--- a/src/server.c
+++ b/src/server.c
@@ -454,7 +454,7 @@ static void show_features (void) {
 #else
       "\t- bzip2 support\n"
 #endif
-#ifdef HAVE_LIBCRYPT
+#if defined(HAVE_CRYPT) || defined(HAVE_CRYPT_R) || defined(HAVE_LIBCRYPT)
       "\t+ crypt support\n"
 #else
       "\t- crypt support\n"
---
